addons/website_forum/static/src/js/website_forum.js

```diff
@@ -265,7 +265,7 @@ publicWidget.registry.websiteForum = publicWidget.Widget.extend({
         ev.preventDefault();
         if (session.is_website_user) {
             this._displayAccessDeniedNotification(
-                Markup`<p>${sprintf(_t('Oh no! Please <a href="%s">sign in</a> to vote'), "/web/login")}</p>`
+                Markup(_t('Oh no! Please <a href=/web/login>sign in</a> to vote</p>'))
```

Comment:
Thank you for picking up this task, @Julien00859 

The idea was to keep the chunk as short as possible to avoid markup errors (small M) during translation; so at least the `p` tags shouldn't be included. (BTW, it looks like you left the opening `p` tag out).
Maybe we do need to nest the `Markups`, then?

Reply:
good catch for that `</p>`

My opinion is that it is actually welcomed by the translation team when they do see the entire string, html included, that our attempts at making multiple translatable pieces that we combine together in a valid html is actually not that productive.

Let's trust our very capable coworkers from the translation team, they are not going to screw-up that html.